Comments/ClientApp/src/components/Header/Header.js

```diff
@@ -22,6 +24,7 @@ export class Header extends PureComponent<PropsType> {
 		const title = this.props.title;
 		const subtitle1 = this.props.subtitle1;
 		const subtitle2 = this.props.subtitle2;
+		const allowRegisterOrganisationLeadLink = this.props.allowRegisterOrganisationLeadLink;
```

Comment:
"destructuring assignment" would be nice here, but it's just personal style: https://medium.com/@lcriswell/destructuring-props-in-react-b1c295005ce0